docs/src/model_structure.md

```diff
@@ -0,0 +1,38 @@
+# ClimaLand model structure
+
+Here, we explain the internal organization of ClimaLand models.
+
+## Standalone and integrated models
+A core concept of ClimaLand's design is the dichotomy of _standalone_ and _integrated_ models.
+In ClimaLand's framework, the land system is divided into individual components: soil, vegetation (or canopy), snow, and surface water (or pond). Each of these components can be run individually (_standalone_) or combined together (_integrated_).
+
+As an example, the soil model `EnergyHydrology` and the canopy model `CanopyModel` can each be set up and run on their own in standalone mode. Or, a user can set up the integrated `SoilCanopyModel`, which internally constructs each of these models and runs them in tandem, exchanging the relevant fluxes between them at each timestep.
+
+## Model variables
+
+All ClimaLand models (standalone and integrated alike) contain two types of variables:
+
+### Prognostic variables
+These variables are solved for at each timestep based on differential equations and timestepping methods implemented in [ClimaTimeSteppers.jl](https://github.com/CliMA/ClimaTimeSteppers.jl). The specific timestepping methods used depend on the model being used, and can be chosen from the many that ClimaTimeSteppers.jl has implemented.
+
+Prognostic variables are stored in the model state, which is represented as the variable `Y`.
+
+For more information about timestepping, please see the timestepping tutorial page (TODO add link).
+
+### Diagnostic variables
+These variables are computed at each timestep as functions of the prognostic variables (as well as other parameters, forcing data, etc).
+
+Prognostic variables are stored in the model cache, which is represented as the variable `p`.
```

Comment:
perhaps we should also introduce the idea of the integrator here

Reply:
I think we can skip this? Especially since the plan is to move away from SciML.